Step_Functions/Preprocessing_All/Bad_Segments.m

```diff
@@ -81,6 +81,7 @@
             if strcmpi(Choice,"ASR")
                 % ASR only works on filtered data. Create copy of data, and
                 % highpassfilter it with 1 Hz
+                % not clear to me where the 1 Hz 'hicutoff' is set. Possibly in another file? Just want to mention it to double check.
```

Comment:
for ASR to work data HAS to be filtered (as stated in line 82). This is done only on a copy of data to identify bad data segments that will be removed. Please remove/update comment